ansible/ansible-ai-connect-service
# AAP-22640: original task names lost in multi-task plays in playbooks (fix)

Jira Issue: <https://issues.redhat.com/browse/AAP-22640>


## Description
Please see the details here: https://issues.redhat.com/browse/AAP-22640

## Testing

### Steps to test
1. Pull down the PR
2. Run the `ai.api.tests.test_formatter` tests to see if the formatter unit tests work.
3. On stage or your local: try to go thru the scenarios mentioned below both for playbooks and roles.

### Scenarios tested
Tests are seperated for roles and playbooks. 

1. Test for an ordinary role
2. Test for a role that has "loop" in it which contains "-name" under it. 
3. Test for a playbook 
4. Test for a playbook that has tasks contains "loop" in it which contains "-name" under it. 

Do all for single and multi task cases.

## Production deployment
